lib/migrations/2_deploy_alchemy.ts

```diff
@@ -0,0 +1,190 @@
+import { Web3 } from "web3";
```

Comment:
There is currently no anticipated use for this file so I think it should be removed from the PR.  We can't publish Arc.js to NPM in a way that only works for Alchemy.